core/domain/platform_parameter_registry.py

```diff
@@ -61,7 +59,7 @@ class Registry:
     @classmethod
     def create_platform_parameter(
         cls,
-        name: enum.Enum,
+        name: ParamNames,
```

Comment:
Shouldn't the type be called ParamName since it's a single name?

Reply:
Well, actually ParamNames represents [platform_parameter_list.ParamNames](https://github.com/oppia/oppia/blob/d1983c7ea30ce4b531922d9a66bd0d191dc5339d/core/domain/platform_parameter_list.py#L26), which includes all the names of platform parameters in the enum type. Though the type means that the name should be one of the names from the enums.
Should I change it to `ParamName`? Just wanted to confirm.
Thanks

Reply:
Yes, I think so -- when something is an int we type it with "int", not "ints". So similar logic applies here.

Reply:
Make sense, updated.
Thanks